specs/deployment-repo-update.md

```diff
@@ -0,0 +1,31 @@
+# Deployment Repo Update Automation
+
+## Summary
+
+The GitOps driven continuous deployment cycle starts with a change in the Git repository that contains resource manifests. The [Flux] provides the
+[Automated Image Update](https://docs.fluxcd.io/en/latest/references/automated-image-update.html) feature that continuously monitors the docker registry and automatically
+updates deployment repo when a new image is released. Obviously that functionality is not available for  Argo CD users.
+Also cannot or don't want to use docker-registry monitoring and only need functionality related to the Git repository update.
+
+This document is meant to collect requirements for the Git repository update functionality. As a next step, we could discuss if it is possible to implement a Golang library or
+a service that can be used in combination with Argo CD and Flux.
```

Comment:
I think it's worth mentioning that Flux is planning to split out the scanning/automation part and that we may be able to reuse that.

Reply:
added a note.